packages/ui/src/proposals/components/ProposalList/ProposalListItem.tsx

```diff
@@ -108,8 +108,8 @@ const CopyButton = styled(CopyButtonTemplate)`
 `
 
 const ProposalItem = styled(TableListItem)`
-  grid-column-gap: 36px;
-  padding-right: 16px;
+  grid-column-gap: 20px;
+  padding-right: 0px;
```

Comment:
Please keep the original padding on the ProposalItem too, so they remain aligned with the headers and also so they don't look crammed.